xbmc/addons/AddonRepos.h

```diff
@@ -167,6 +168,7 @@ class CAddonRepos
    */
   bool FindDependencyByParentRepo(const std::string& dependsId,
                                   const std::shared_ptr<IAddon>& parent,
+                                  const std::string& parentRepoId,
```

Comment:
Think you need then also to add the doc about above :smirk:

Reply:
yes